scala3doc/src/main/scala/dotty/dokka/DottyDokkaConfig.scala

```diff
@@ -19,7 +19,8 @@ case class DottyDokkaConfig(docConfiguration: DocConfiguration) extends DokkaCon
 
   private object OurConfig extends DokkaConfiguration.PluginConfiguration:
     override def getFqPluginName = "ExternalDocsTooKey"
-    override def getSerializationFormat: DokkaConfiguration$SerializationFormat = DokkaConfiguration$SerializationFormat.JSON
+    override def getSerializationFormat: DokkaConfiguration$SerializationFormat =
+      DokkaConfiguration$SerializationFormat.JSON.asInstanceOf[DokkaConfiguration$SerializationFormat]
```

Comment:
Why do need to cast here? Some change in dotty type interference?

Reply:
This became necessary when I bumped up Dotty compiler version to latest. I'm not sure if this is related to type inference, my first guess is that Kotlin was emitting some strange JVM bytecode and we changed how we translated that to Scala types. Another of the litany of things we should open an issue for in the Dotty repo.

Reply:
ok, I will take a look